docs/resiliency/index.asciidoc

```diff
@@ -154,6 +140,20 @@ shard.
 
 == Completed
 
+[float]
+=== The _version field may not uniquely identify document content during a network partition (STATUS: DONE, v7.0.0)
```

Comment:
Status: DONE. Can you prove it? ;)
Should we only move this after more extensive testing?